src/connection.cpp

```diff
@@ -23,7 +23,7 @@ ConnectionLoader::ConnectionLoader(MainWindow* main, RPC* rpc) {
     QMovie *movie2 = new QMovie(":/img/res/silentdragon-animated-startup-dark.gif");;
     auto theme = Settings::getInstance()->get_theme_name();
     auto size  = QSize(512,512);
-    if (theme == "dark") {
+    if (theme == "dark" || "midnight") {
```

Comment:
if you search src/*.cpp you will see there are 2 other places that need this same change

Reply:
Okay, dokey; working on this now. Thanks for the review @leto